pkgs/development/python-modules/maturin/default.nix

```diff
@@ -0,0 +1,41 @@
+{ buildPythonPackage
+, maturin
+, setuptools
+, toml
+, wheel
+, isPy3k
+}:
+
+buildPythonPackage {
+  inherit (maturin) pname version src meta;
+
+  format = "pyproject";
+
+  nativeBuildInputs = [
+    setuptools
+    toml
+    wheel
+  ];
+
+  propagatedBuildInputs = [
```

Comment:
probably better not to propagate and let packages add their own versions of rustc and cargo to the derivation

Reply:
I also think it'd be clearer if user packages bring their own rust programs explicitly but not sure what's the norm there.